src/com/osudroid/ui/v2/multi/RoomPlayerButton.kt

```diff
@@ -0,0 +1,172 @@
+                }
+
+                missingIndicator = sprite {
+                    textureRegion = ResourceManager.getInstance().getTexture("missing")
+                    anchor = Anchor.CenterLeft
+                    origin = Anchor.CenterLeft
+                    size = Vec2(18f)
+                }
+            }
+
+            +ModsIndicator().apply {
+                minHeight = 18f // Force to take space even if no mods are enabled
+                iconSize = 18f
+                modsIndicator = this
+            }
+        }
+
+    }
+
+
+    fun updateState(room: Room, player: RoomPlayer) {
+
+        background!!.apply {
+            color = when {
+                room.isTeamVersus -> when (player.team) {
+                    Blue -> Color4("#A0C0FF") * 0.1f
+                    Red -> Color4("#FFA0A0") * 0.1f
+                    null -> Theme.current.accentColor * 0.1f
+                }
+
+                else -> Theme.current.accentColor * 0.1f
+            }
+            alpha = 0.5f
+        }
+
+        foreground!!.color = when (player.status) {
+            Playing -> Theme.current.accentColor
+            Ready -> Color4("#A0FFA0")
+            NotReady, MissingBeatmap -> Color4("#FFA0A0")
+        }
+
+        nameText.text = player.name
+        missingIndicator.isVisible = player.status == MissingBeatmap
+        modsIndicator.mods = if (room.gameplaySettings.isFreeMod) player.mods.json else null
+
+        onActionLongPress = {
+            UIDropdown(this@RoomPlayerButton).apply dropdown@{
+                maxWidth = 260f
+
+                addButton {
+                    setText(R.string.multiplayer_room_player_menu_view_profile)
+                    onActionUp = {
+                        WebViewFragment()
+                            .setURL(WebViewFragment.PROFILE_URL + player.id)
+                            .show()
+                        this@dropdown.hide()
+                    }
+                }
+
+                if (player.id != Multiplayer.player!!.id) {
+                    addButton {
+                        setText(if (player.isMuted) R.string.multiplayer_room_player_menu_unmute else R.string.multiplayer_room_player_menu_mute)
+                        onActionUp = {
+                            player.isMuted = !player.isMuted
+                            this@dropdown.hide()
+                        }
+                    }
+
+                    if (Multiplayer.isRoomHost) {
+                        addButton {
+                            setText(R.string.multiplayer_room_player_menu_transfer_host)
+                            onActionUp = {
+                                UIConfirmDialog().apply {
+                                    title = StringTable.get(R.string.multiplayer_room_player_transfer_dialog_title)
+                                    text = StringTable.format(R.string.multiplayer_room_player_transfer_dialog_message, player.name)
```

Comment:
Player name is in `title`, not `text`.
```suggestion
                                    title = StringTable.format(R.string.multiplayer_room_player_transfer_dialog_title, player.name)
                                    text = StringTable.get(R.string.multiplayer_room_player_transfer_dialog_message)
```